robot_smach_states/src/robot_smach_states/util/designators/arm.py

```diff
@@ -73,7 +73,7 @@ class UnoccupiedArmDesignator(ArmDesignator):
     >>>
```

Comment:
@LoyVanBeek I merged master into this branch since I couldn't merge due to a conflict in `designators/arm.py`. I accepted 'remote' on each occasion so not many changes left